lib/mindwendel_web/templates/layout/root.html.leex

```diff
@@ -23,23 +23,23 @@
           data-bs-target="#navbarDropdownToggler"
           aria-controls="navbarDropdownToggler"
           aria-expanded="false"
-          aria-label="Toggle navigation" >
+          aria-label="Toggle navigation">
           <span class="navbar-toggler-icon"></span>
         </button>
 
         <div class="collapse navbar-collapse" id="navbarDropdownToggler">
-          <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+          <ul class="navbar-nav me-auto mb-2 mb-lg-0 bg-white">
             <li class="nav-item">
               <a class="nav-link" aria-current="page" href="/">New brainstorming</a>
             </li>
 
             <%= if assigns[:current_user] do %>
-              <li class="nav-item dropdown d-none d-lg-block">
+              <li class="nav-item dropdown d-none d-lg-block bg-white">
                 <a class="nav-link dropdown-toggle" data-bs-toggle="dropdown" href="#" role="button" aria-expanded="false">
                   Your brainstormings
                 </a>
                 <ul class="dropdown-menu">
-                  <%= for brainstorming <- @current_user.brainstormings do %>
+                  <%= for brainstorming <- list_brainstormings_for(@current_user, 10) do %>
```

Comment:
Normally, when following a MVC pattern, I would not initiate a database call in the view (or in `MindwendelWeb.LayoutView`). What do you say we move this to the controller?

Reply:
there is no controller for the root layout, because this is used inside the live views. You could maybe move to the mount method of the most upper live view, but I am not really convinced this is the way to go.